### ISSUE TITLE
setDisplayName doesn't change anything

### ISSUE THREAD

**Issue by salzteam** [pedroslopez/whatsapp-web.js#2541]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I tried changing my name using this code: 

    if (msg.startsWith('setname') || msg.startsWith('Setname') || msg.startsWith('SETNAME')) {
      const new_name = msg.split('|')[1];
      await client.setDisplayName(new_name);
      message.reply(`Success change name to ${new_name}`);
    }
    
  but response from client.setDisplayName(new_name) is false and doesn't change anything

### Expected behavior

as a use, i expected to change my display name

### Steps to Reproduce the Bug or Issue

-

### Relevant Code

-

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

-

### Additional context

-

**Comment by tofers** [comment]
#2295  fix method

**Comment by salzteam** [comment]
> #2295 fix method

status response is true, but myname still not update, i already update my Inject and Clients like your commit on "[fixing the setPushname function](https://github.com/pedroslopez/whatsapp-web.js/pull/2295/commits/8622a1cd14bacf59022624cd103de0d83de6531c)"

**Comment by ShanthiniMichael** [comment]
> > #2295 fix method
> 
> status response is true, but myname still not update, i already update my Inject and Clients like your commit on "[fixing the setPushname function](https://github.com/pedroslopez/whatsapp-web.js/pull/2295/commits/8622a1cd14bacf59022624cd103de0d83de6531c)"

Have you found any solution?

**Comment by salzteam** [comment]
@alechkos 
I have carried out your instructions, but the result from console given are still false,  why is that?

```
const result = await client.setDisplayName(new_name);
console.log(result);
```

**Comment by alechkos** [comment]
@salzteam 
update the lib to latest version

**Comment by salzteam** [comment]
@alechkos 

use
```
 npm i github:pedroslopez/whatsapp-web.js
```
or
```
npm i github:alechkos/whatsapp-web.js
```

**Comment by salzteam** [comment]
@alechkos 

work after i add like this
```
    async setDisplayName(displayName) {
        const couldSet = await this.pupPage.evaluate(async displayName => {
            // if(!window.Store.Conn.canSetMyPushname()) return false;

            if(window.Store.MDBackend) {
                await window.Store.Settings.setPushname(displayName);
                return true;
            } else {
                const res = await window.Store.Wap.setPushname(displayName);
                return !res.status || res.status === 200;
            }
        }, displayName);

        return couldSet;
    }
```

**Comment by alechkos** [comment]
@salzteam 
> work after i add like this in src/Client.js

you don't have to do that
you have to update the lib to the latest version, that's all lol
```js
npm i whatsapp-web.js@1.23.1-alpha.0
```

**Comment by salzteam** [comment]
@alechkos 
I've updated the lib, but I don't know why if I don't add it like that, the code doesn't work, do I need to reconnect qr again?

**Comment by alechkos** [comment]
Fixed in #2295
